src/aws-sdk/sdk.ts

```diff
@@ -58,6 +58,55 @@ class ServiceProxy {
   }
 }
 
+/**
+ * Step Functions AWS Service Integrations support integrations with many services, however, it has a list of
+ * services or methods that are not supported.
+ *
+ * https://docs.aws.amazon.com/step-functions/latest/dg/supported-services-awssdk.html
+ */
```

Comment:
What happens when they add support for one? Are we then blocking users?

Reply:
Yeah, we could add an option to not validate.

Reply:
Maybe these should be warnings, not blocking errors?